Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageFailedExEventArgs.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    public delegate void ImageFailedExEventHandler(object sender, ImageFailedExEventArgs e);
+
+    public class ImageFailedExEventArgs : EventArgs
+    {
+        public ImageFailedExEventArgs(Exception errorException)
+        {
+            ErrorException = errorException;
+            if (ErrorException != null)
+            {
+                ErrorMessage = ErrorException.Message;
+            }
+            else
+            {
+                ErrorMessage = null;
```

Comment:
This doesn't need setting. You're already checking to see if it's null, getting to this point would mean that ErrorMessage was already null. You could simplify even further and do away with the if and just say

ErrorMessage = ErrorException?.Message;